ci/infra/azure/README.md

```diff
@@ -0,0 +1,160 @@
+# Introduction
+
+This terraform project creates the infrastructure needed to run a
+cluster on top of Azure.
+
+Once the infrastructure is up and running nothing special has to be done
+to deploy CaaS Platform on top of it.
+
+This document focuses on the key aspects of the infrastructure created
+by terraform.
+
+# Cluster layout
+
+## Machines
+
+As usual the cluster is based on two types of nodes: master and worker nodes.
+
+All the nodes are created using the SLES 15 SP1 container host image built
```

Comment:
SLES 15 SP2 (it can be updated later on)

Reply:
Currenly, we do not have SP2 on Azure.  Maybe we should revisit this next week and verify.

Reply:
You could use GMC images that have been tested by QA
https://portal.azure.com/#@SUSERDBillingsuse.onmicrosoft.com/resource/subscriptions/c011786b-59d7-4817-880c-7cd8a6ca4b19/resourceGroups/openqa-upload/providers/Microsoft.Compute/disks/SLES15-SP2-BYOS.x86_64-0.9.13-Azure-Build1.6.vhd

Reply:
Trying to follow the guide to  and try setup using [blob_uri](https://www.terraform.io/docs/providers/azurerm/r/image.html#blob_uri), when I use command `az image list -g OPENQA-UPLOAD` and found

```
  {
    "hyperVgeneration": "V1",
    "id": "/subscriptions/c011786b-59d7-4817-880c-7cd8a6ca4b19/resourceGroups/OPENQA-UPLOAD/providers/Microsoft.Compute/images/SLES15-SP2-CHOST-BYOS.x86_64-0.9.12-Azure-Build1.4.vhd",
    "location": "westeurope",
    "name": "SLES15-SP2-CHOST-BYOS.x86_64-0.9.12-Azure-Build1.4.vhd",
    "provisioningState": "Succeeded",
    "resourceGroup": "OPENQA-UPLOAD",
    "sourceVirtualMachine": null,
    "storageProfile": {
      "dataDisks": [],
      "osDisk": {
        "blobUri": null,
        "caching": "None",
        "diskEncryptionSet": null,
        "diskSizeGb": 30,
        "managedDisk": {
          "id": "/subscriptions/c011786b-59d7-4817-880c-7cd8a6ca4b19/resourceGroups/openqa-upload/providers/Microsoft.Compute/disks/SLES15-SP2-CHOST-BYOS.x86_64-0.9.12-Azure-Build1.4.vhd",
          "resourceGroup": "openqa-upload"
        },
        "osState": "Generalized",
        "osType": "Linux",
        "snapshot": null,
        "storageAccountType": "Standard_LRS"
      },
      "zoneResilient": null
    },
    "tags": {},
    "type": "Microsoft.Compute/images"
  },
```

Seems like blob_uri is not setup.  @pgonin Anyway we can set it up?